src/test/testLogger.ts

```diff
@@ -3,8 +3,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { appendFileSync } from 'fs-extra'
+import { join } from 'path'
 import { Loggable, Logger, LogLevel } from '../shared/logger'
 
+export const testLogOutput = join(__dirname, '..', '..', '..', '.test-reports', 'testLog.log')
```

Comment:
there is no reason to use join() on string literals. `/` slashes are supported everywhere (except some cmd.exe shell tools, but this path isn't being send to cmd.exe).

```suggestion
export const testLogOutput = join(__dirname, '../../../.test-reports/testLog.log')
```

Reply:
Agreed, the project does this a lot and we should probably stop